src/tutorial/adding-an-api.md

```diff
@@ -0,0 +1,40 @@
+# Adding an API
+
+Each Denops plugin registers one or more functions as APIs to Denops.
+First, try to write an `echo()` function that returns a given string and register it as an API.
+You can rewrite `main.ts` as follows:
+
+```ts:main.ts
+import { Denops } from "https://deno.land/x/denops_std@v1.0.0-beta.2/mod.ts";
+import { ensureString } from "https://deno.land/x/unknownutil@v0.1.1/mod.ts";
+
+export async function main(denops: Denops): Promise<void> {
+  denops.dispatcher = {
+    async echo(text: unknown): Promise<unknown> {
+      // assure `text` is string type.
+      ensureString(text);
+      return await Promise.resolve(text);
+    },
+  };
+};
+```
+
+Note that you can register a function that satisfies the following as an API:
+- All of its arguments must be `unknown`.
```

Comment:
may

```suggestion
- All of its argument types must be `unknown`.
```

Reply:
I thing it would be better to use "of type" for type restrictions, so revised in this way at af3096c.